edu/jalc/automobile/onlinebuilder/builders/dodgeram/ram1500/builders/Ram1500RebelBuilder.java

```diff
@@ -0,0 +1,229 @@
+      EngineAssembly vvt_Engine = new StandardEcoEngine(vvtEngine, new EconomyExhaust(), new NaturallyAspiratedInduction());
+
+      promptBuilder.addType("Engine ");
+      promptBuilder.addOption(hemi_VVTEngine);
+      promptBuilder.addOption(vvt_Engine);
+      promptBuilder.sort();
+      int engineChoice = 2;
+      try{
+         engineChoice = promptBuilder.build().ask();
+      }
+
+      catch(Exception e){}
+      engine = (EngineAssembly)promptBuilder.getOptions().get(engineChoice - 1);
+
+       //axle choice
+      int axleChoice = 1;
+      TerminalPrompterBuilder axlePrompter = TerminalPrompterBuilder.newBuilder();
+      try{
+         axleChoice = axlePrompter.addType("Axle")
+            .addOption(new TruckRearAxle(3.21))
+            .addOption(new TruckRearAxle(3.92))
+            .sort()
+            .build()
+            .ask();
+      }
+      catch(Exception e){System.err.println(e);}
+      truckRearAxle = (TruckRearAxle)axlePrompter.getOptions().get(axleChoice - 1);
+
+      return  this;
+   }
+
+   public TruckDodgeRamBuilderInterface askForColorAndInterior(){
+
+     //exterior paint choice
+      TerminalPrompterBuilder paintPrompter = TerminalPrompterBuilder.newBuilder();
+      int choice = 1 ;
+      paintPrompter.addType("Paint")
+         .addOption(new FlameRedClearCoat())
+         .sort();
+
+      try{
+         choice = paintPrompter.build().ask();
+      }
+      catch(Exception e){System.err.println(e);}
+
+      paint = (Paint)paintPrompter.getOptions().get(choice - 1);
+
+      //Seat choice
+      TerminalPrompterBuilder seatPrompter = TerminalPrompterBuilder.newBuilder();
+      int seatChoice = 1;
+      try{
+         seatChoice = seatPrompter.addType("Seats")
+            .addOption(new PremiumClothSeatsWithEmbossedTreadPattern())
+            .build()
+            .ask();
+      }
+      catch(Exception e){System.err.println(e);}
+      seat = (TruckSeat)seatPrompter.getOptions().get(seatChoice - 1);
+
+      this.body = new CrewCab(
+         new Quarterpanels(paint,null),
+         new EngineCompartment(new Hood(paint, null)),
+         new TruckCabin(seat),
+            cabAndBox);
+      return this;
+   }
+
+   public TruckDodgeRamBuilderInterface askForOptions(){
+
+      try{
+         TerminalPrompterBuilder wheelPrompter = TerminalPrompterBuilder.newBuilder();
+         int wheelChoice = wheelPrompter.addType("Wheels")
+            .addOption(new TruckAliminumWheel(17.0, 8.0))
+            .addOption(new TruckAlumWheel(17.0, 8.0))
+            .sort()
+            .build()
+            .ask();
+         wheel = (TruckWheel)wheelPrompter.getOptions().get(wheelChoice - 1);
+      }
+      catch(Exception e){System.err.println(e);}
+
+      try{
+         TerminalPrompterBuilder tirePrompter = TerminalPrompterBuilder.newBuilder();
+         int tireChoice = tirePrompter.addType("Tires")
+            .addOption(new BSWAllSeasonTire("P285/70R17"))
```

Comment:
don't worry about it , I will fix it since it's a small mistake, just to get it out of the way  and merge everything!